### PR TITLE
BOAC-5286, BOAC-5287. BOAC-5306, BOAC-5398 - various fixes for notes and passenger manifest

### PR DESCRIPTION
**Pull request by mohamalnadi**
Jiras:

- https://jira-secure.berkeley.edu/browse/BOAC-5286
- https://jira-secure.berkeley.edu/browse/BOAC-5287
- https://jira-secure.berkeley.edu/browse/BOAC-5306
- https://jira-secure.berkeley.edu/browse/BOAC-5398

### CODE REVIEW

File: src/components/admin/Users.vue [hunk 1]
```diff
@@ -511,9 +511,11 @@ export default {
   },
   methods: {
     handleSort(sortBy) {
-      this.sortBy = sortBy[0].key
-      this.sortDesc = sortBy[0].order === 'asc' ? false : true
-      this.usersProvider() // Method to fetch users with new sorting parameters
+      if (sortBy.length) {
+        this.sortBy = sortBy[0]?.key
+        this.sortDesc = sortBy[0]?.order === 'asc' ? false : true
+        this.usersProvider() // Method to fetch users with new sorting parameters
```
Comment (johncrossman) [inline src/components/admin/Users.vue:517]: @mohamalnadi - I'm concerned with the null-checks here. Are we getting a non-empty `sortBy` array containing nulls? If so, perhaps we have a problem upstream from this code. Is the input to this function coming from `v-data-table`? As before, we should be very cautious with null-tolerant question-marks.

File: src/components/admin/Users.vue [hunk 2]
```diff
@@ -847,4 +849,8 @@ export default {
   position: relative;
   display: inline-block;
 }
+
+:deep(.v-table > .v-table__wrapper > table > thead > tr > th) {
+  height: 0px !important;
+}
```
Comment (johncrossman) [inline src/components/admin/Users.vue:855]: @mohamalnadi - Formatting of `th` elements (table-header cells in `v-data-table`) can be done using the `header-props` component prop. Elaborate CSS is often necessary but, in this case, I'd suggest using `header-props`.

File: src/components/note/AdvisingNote.vue [hunk 3]
```diff
@@ -309,4 +313,10 @@ loadAuthorDetails()
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+.ellipses-for-long-text {
+  width: 100%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
```
Comment (johncrossman) [inline src/components/note/AdvisingNote.vue]: @mohamalnadi - Minor indentation fix needed above. Thx.

File: src/components/search/AdvancedSearchModal.vue [hunk 4]
```diff
@@ -317,6 +319,17 @@ const cancel = () => {
   setTimeout(reset, 100)
 }
 
+const focusOnDate = (type) => {
+  if (type === 'from') {
+    const element = document.getElementById('search-options-note-filters-last-updated-from')
+    element.focus()
+  } else if (type === 'to') {
+    const element = document.getElementById('search-options-note-filters-last-updated-to')
+    element.focus()
```
Comment (johncrossman) [inline src/components/search/AdvancedSearchModal.vue]: @mohamalnadi - Thank you for considering accessibility here. Fyi, we have `putFocusNextTick` (utility function) that could be used in place of the raw script above.

**Comment by johncrossman** [comment]
Thank you @mohamalnadi. I've added a few comments/questions. Let me know if anything is unclear.

File: src/components/admin/Users.vue (hunk 1 above)
Comment (mohamalnadi) [reply src/components/admin/Users.vue:517]: @johncrossman - yes, I just removed the null-checks, they technically do not do anything as that part of the code only runs if sortBy has a length of more than 0.

File: src/components/admin/Users.vue (hunk 2 above)
Comment (mohamalnadi) [reply src/components/admin/Users.vue:855]: This CSS is only relevant when the table is in a loading state. I do not think header-props will work in this case.

File: src/components/note/AdvisingNote.vue (hunk 3 above)
Comment (mohamalnadi) [reply src/components/note/AdvisingNote.vue]: fixed!

File: src/components/search/AdvancedSearchModal.vue (hunk 4 above)
Comment (mohamalnadi) [reply src/components/search/AdvancedSearchModal.vue]: replaced with `putFocusNextTick `, thank you!!

**Comment by mohamalnadi** [comment]
@johncrossman - please see my updated changes and comments, thank you for you feedback!

File: src/components/admin/Users.vue (hunk 2 above)
Comment (johncrossman) [reply src/components/admin/Users.vue:855]: Got it. Thanks @mohamalnadi

**Comment by johncrossman** [comment]
Thank you @mohamalnadi

**Pull request merged**